### ISSUE TITLE
Inconsistent max reputation

### ISSUE THREAD

**Issue by Pimhazeveld** [nroutasuo/level13#129]
**Game version**
0.2.4

**Browser**
Chrome

**Description**
I have the exact same buildings on 2 different camps but different reputation max. Both camps have 50% population. Is the reputation max based on the amount of raids the camp got or something?

[Camp 1](https://i.imgur.com/KKGsqph.png)

[Camp 2](https://i.imgur.com/pwbzxHh.png)

**Comment by Brilliand** [comment]
Note that one of them has 37 reputation from buildings, while the other has 36.  I don't know why that's different, but it might explain why one has a "Poor Defenses" penalty and the other doesn't.

**Comment by Pimhazeveld** [comment]
> Note that one of them has 37 reputation from buildings, while the other has 36. I don't know why that's different, but it might explain why one has a "Poor Defenses" penalty and the other doesn't.

That's weird because both have the same amount of buildings....

That is... I might have abused some storage glitch and built a hut on another non-camp tile in one of the levels and that may somehow affect reputation of the main camp.

**Comment by Brilliand** [comment]
I have the same mysterious variation in my camp reputations, and I haven't
abused that glitch.  Maybe it's a lingering effect of the disappearing
clinics?

**Comment by Pimhazeveld** [comment]
> I have the same mysterious variation in my camp reputations, and I haven't abused that glitch. Maybe it's a lingering effect of the disappearing clinics?

It could be actually. I looked closely at the part where I used the glitch and the camp where I used it on has more reputation than the one I didn't use it on. I also built a camp on the one which I didn't build on yet and it didn't affect reputation.

The camp that had more reputation didn't exist until the update which magically removed the clinics. however none of the other camps seem to have inconsistencies.
